### DISCUSSION TITLE
Kubb Best practices documentation

### DISCUSSION THREAD

**Discussion by Samuel-Morgan-Tyghe** [kubb-labs/kubb#1063] [Ideas]
It would be great to see best practices for kubb generation, demos - example projects.
questions I would love to hear answered

1. How to scale?
2. How to quickly navigate / make sense of generated code?
3. When not to use Kubb

**Comment by stijnvanhulle** [comment]
Could this help: https://kubb.dev/knowledge-base/best-practices
